cmd/krel/cmd/changelog.go

```diff
@@ -93,9 +92,8 @@ const (
 
 func init() {
 	const (
-		tagFlag   = "tag"
-		tarsFlag  = "tars"
-		tokenFlag = "token"
+		tagFlag  = "tag"
+		tarsFlag = "tars"
```

Comment:
Unrelated to the PR, but is there a reason for these?
Could we just fix this in `changelogCmd.PersistentFlags().StringVar`?

Reply:
No, I'll remove it.